.gitmodules

```diff
@@ -1,3 +1,6 @@
 [submodule "riscv-arch-test"]
 	path = tests/riscv-arch-test
 	url = https://github.com/riscv-non-isa/riscv-arch-test
+[submodule "mini-gdbstub"]
+	path = mini-gdbstub
+	url = https://github.com/RinHizakura/mini-gdbstub
```

Comment:
You should relicense `mini-gdbstub` in MIT License or similar permissive terms.

Reply:
Otherwise, we might encounter potential license conflicts if we are about to link statically.

Reply:
`mini-gdbstub` is licensed to MIT license now